src/components/Offers/Edit/EditOfferForm.js

```diff
@@ -0,0 +1,155 @@
+import React, { useCallback, useContext, useEffect } from "react";
+import { parseRequestErrors } from "../Form/OfferUtils";
+import OfferForm from "../Form/form-components/OfferForm";
+import { editOffer } from "../../../services/offerService";
+import { Redirect, useLocation, useParams } from "react-router-dom";
+import useOffer from "../../../hooks/useOffer";
+import useOfferForm from "../../../hooks/useOfferForm";
+import { INITIAL_JOB_DURATION } from "../../../reducers/searchOffersReducer";
+import useSession from "../../../hooks/useSession";
+
+export const EditOfferControllerContext = React.createContext();
+
+function parseDescription(description) {
+    const temp = document.createElement("span");
```

Comment:
remove the DOM element after this

Reply:
I didn't know it was added to the dom without adding as some child

Reply:
Hmm good question, maybe it doesn't